Example/UICatalog.xcodeproj/xcshareddata/xcschemes/UICatalog.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0800"
+   LastUpgradeVersion = "0930"
```

Comment:
We will want to keep this for Xcode 8 for now. So you can revert the changes to this file.